example/pubspec.yaml

```diff
@@ -1,6 +1,9 @@
 name: wifi_iot_example
 description: Demonstrates how to use the wifi_iot plugin.
 
+environment:
+  sdk: '>=2.10.0 <3.0.0'
```

Comment:
To clarify is it `2.10.0` or `2.12.0`? since the null-safety are available in version `>= 2.12.0` of dart.

Reply:
example code is not null-safe yet.

Reply:
example app barely works, needs some work.

Reply:
I'll contribute.

Reply:
LGTM.. I'll make a new PR once this PR merged.